.gitignore

```diff
@@ -41,6 +42,7 @@ test/.wct-kludge
 # IDEs
 .vimrc
 .sonarlint
+.history
```

Comment:
Love this addition!  We'll only need it once though and it seems to appear on both line 29 & 45.  Either one is great.

Reply:
@castastrophe a merge error I suspect, pushing an update shortly